test/regression.uts

```diff
@@ -5466,10 +5467,11 @@ assert pkt.nscount == 0
 assert pkt.ancount == 0
 
 = Answers - building
-a = UDP()/LLMNRResponse()
-b = UDP()/LLMNRQuery()
+a = UDP()/LLMNRResponse(id=12)
```

Comment:
Is this needed ?

Reply:
Well this checks that the `answers` works properly with `id` (https://github.com/secdev/scapy/blob/master/scapy/layers/llmnr.py#L46-L49)

Reply:
Thanks !